src/views/Admin/AdminNetworkingKeys.js

```diff
@@ -65,54 +76,56 @@ function useSetKeys() {
     }
   }, [_point, confirmed, syncOwnedPoint]);
 
-  const setKeys = useCallback(async () => {
-    const details = need.details(getDetails(_point));
-    const networkRevision = parseInt(details.keyRevisionNumber, 10);
-
-    console.log('setting revision to ', networkRevision + 1);
+  const construct = useCallback(
+    async (manualSeed, isDiscontinuity) => {
+      const details = need.details(getDetails(_point));
+      const networkRevision = parseInt(details.keyRevisionNumber, 10);
+
+      let _seed = manualSeed;
+      if (!_seed) {
+        const networkSeed = await attemptNetworkSeedDerivation({
+          urbitWallet,
+          wallet,
+          authMnemonic,
+          details,
+          revision: networkRevision + 1,
+        });
+
+        _seed = networkSeed.matchWith({
+          Nothing: () => {
+            const ndSeed = randomHex(64);
+            setNdNetworkSeed(ndSeed);
+            return ndSeed;
+          },
+          Just: p => p.value,
+        });
+      }
+      const pair = deriveNetworkKeys(_seed);
+
+      const txn = azimuth.ecliptic.configureKeys(
+        _contracts,
+        _point,
+        addHexPrefix(pair.crypt.public),
+        addHexPrefix(pair.auth.public),
+        CRYPTO_SUITE_VERSION,
+        isDiscontinuity ? 1 : 0
```

Comment:
`true : false` (and probably also just `isDiscontinuity`) should work here, right? Why is this written like this?

Reply:
ah, thought it had to be a number — sgtm